imagepuller/internal/service/verify.go

```diff
@@ -84,52 +85,32 @@ func (s *ImagePullerService) getAndVerifyImage(ctx context.Context, log *slog.Lo
 	return remoteImg, nil
 }
 
-func (s *ImagePullerService) storeAndVerifyLayers(log *slog.Logger, remoteImg gcr.Image) (string, error) {
+func (s *ImagePullerService) storeAndVerifyLayers(log *slog.Logger, remoteImg gcr.Image) error {
```

Comment:
The name is not really fitting anymore, since we don't do manual digest checks any longer.

Maybe this should even be moved into the new `store` package, as `store.Store(gcr.Image)` or `store.StoreDeduplicated(...)`?